### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by Cashiot**
Further spanish translation to match the game content

### CODE REVIEW

File: application/src/main/resources/locale/blueprint/horizons/description.csv [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
+blueprint.description.horizons.engineer_a3c_learn,Learn about: Reach level 3 and 33% reputation with Marco Qwent.,Learn about: Reach level 3 and 33% reputation with Marco Qwent.,Erfahren Sie mehr über: Ruf der Stufe 3 und 33% mit Marco Qwent.,Descubrimiento: Alcanzar reputación de nivel 3 y 33% con Marco Qwent.,Découverte: Atteindre la réputation de niveau 3 et 33% avec Marco Qwent.,Aprenda sobre: Alcance Nível 3 e 33% de reputação com Marco Qwent.,Узнайте о репутации уровня 3 и 33% с Марко Квентом.
+blueprint.description.horizons.engineer_a3c_unlock,Initial contract: 25 units of Sensor Fragments.,Initial contract: 25 units of Sensor Fragments.,Erstvertrag: 25 Einheiten von Sensorfragmenten.,"Contrato inicial: 25 unidades de fragmentos de sensor: se encuentran visitando HIP 14479 y buscando fuentes de Señal de Origen no Humano [amenaza 4] "". En estos aparecerá un Sensor Thargoide flotando en el espacio. Destruyelo y recolecta los fragmentos. Cada trozo que recogas contiene 3 fragmentos de sensor Thargoide "".",Contrat initial: 25 unités de fragments de capteur.,Contrato inicial: 25 unidades de fragmentos de sensores.,Первоначальный контракт: 25 единиц фрагментов датчиков.
```
Comment (jixxed) [inline application/src/main/resources/locale/blueprint/horizons/description.csv:19]: oddly many double quotes in here

Comment (jixxed) [reply application/src/main/resources/locale/blueprint/horizons/description.csv:19]: looks like it also happened in a previous PR

**Review by jixxed — CHANGES_REQUESTED** [review]
please check the double quotes

**Comment by Cashiot** [comment]
I think that should be sorted out by now. Only one word has been left in quotation marks because that is how it appears in the original description.

I don't think that's the case, but I don't know if I've deleted something that made sense in other languages, because it wasn't only Spanish that had these problems.

**Comment by jixxed** [comment]
thnx for checking 👍

**Pull request merged**